From: Bruce Rogers <brogers@suse.com>
Date: Tue, 20 Nov 2018 15:46:41 -0700
Subject: tests/qemu-iotests: Triple timeout of i/o tests due to obs
 environment

Executing tests in obs is very fickle, since you aren't guaranteed
reliable cpu time. Triple the timeout for each test to help ensure
we don't fail a test because the stars align against us.

Signed-off-by: Bruce Rogers <brogers@suse.com>
[DF: Small tweaks necessary for rebasing on top of 6.2.0]
Signed-off-by: Dario Faggioli <dfaggioli@suse.com>
---
 tests/qemu-iotests/common.qemu | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/qemu-iotests/common.qemu b/tests/qemu-iotests/common.qemu
index 0f1fecc68edfb07633589b0f3c75..b23c4798aa112efee49cf5c9469e 100644
--- a/tests/qemu-iotests/common.qemu
+++ b/tests/qemu-iotests/common.qemu
@@ -85,7 +85,7 @@ _timed_wait_for()
     timeout=yes
 
     QEMU_STATUS[$h]=0
-    read_timeout="-t ${QEMU_COMM_TIMEOUT}"
+    read_timeout="-t $((${QEMU_COMM_TIMEOUT}*3))"
     if [ -n "${GDB_OPTIONS}" ]; then
         read_timeout=
     fi
